lib/common.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+export SEP="====================================================================="
+
+log(){
+  echo "$(date +%Y-%m-%dT%H:%M:%S%z) INFO: $(basename "${0}") [$$] ${*}"
```

Comment:
Spacing is inconsistent in shell scripts - it's 2 spaces in some places, 4 spaces in others

Reply:
fixed file spacing